java/java.source.base/src/org/netbeans/modules/java/source/usages/ExecutableFilesIndex.java

```diff
@@ -164,7 +164,7 @@ static String wrap(Set<String> values) {
                 attribute.append("::"); //NOI18N
             }
             s = s.replaceAll("\\\\", "\\\\\\\\"); //NOI18N
-            s = s.replaceAll(":", "\\\\d"); //NOI18N
+            s = s.replace(":", "\\\\d"); //NOI18N
```

Comment:
Hmmmm, the replacement string is *probably* not equivalent (I think it should have 1/2 of the '\', but please check).